syntax = "proto3";
package Command_BUTTON;

message PRESS {
  uint32 uid = 1;
  string function = 2;
};
